xml/templates/civicrm_data.tpl

```diff
@@ -1652,6 +1655,7 @@ SELECT @option_value_rel_id_exp  := value FROM civicrm_option_value WHERE option
 SELECT @option_value_rel_id_ar  := value FROM civicrm_option_value WHERE option_group_id = @option_group_id_arel AND name = 'Accounts Receivable Account is';
 SELECT @option_value_rel_id_as  := value FROM civicrm_option_value WHERE option_group_id = @option_group_id_arel AND name = 'Asset Account is';
 SELECT @option_value_rel_id_cg  := value FROM civicrm_option_value WHERE option_group_id = @option_group_id_arel AND name = 'Cost of Sales Account is';
+SELECT @option_value_rel_id_dr  := value FROM civicrm_option_value WHERE option_group_id = @option_group_id_arel AND name = 'Deferred Revenue Account is';
```

Comment:
My comments seem to be on different lines now than I had intended. As noted in the issue, the comments are with respect to localization for description fields, not name fields.